ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt

```diff
@@ -18,7 +19,9 @@ import java.awt.event.KeyEvent
 import java.time.Duration
 
 fun RemoteRobot.idea(function: IdeaFrame.() -> Unit) {
-    find<IdeaFrame>().apply(function)
+    val frame = find<IdeaFrame>()
+    frame.apply { tryCloseTips() }
```

Comment:
Looks like they made this a system property for 203+ (https://github.com/JetBrains/intellij-community/commit/d23411df268f9de5e5311920e8bfe529285709e8)

Oh well..

Reply:
Added our first FIX_WHEN_MIN_IS_203 :D

Reply:
We don't even have a 202 build yet /cry